repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,147 @@
+{
+  "type": "object",
+  "properties": {
+    "marathon/libprocess-ip": {
+      "description": "Interface to bind to for communication with the Mesos master.",
+      "type": "string"
+    },
+    "mesos/master": {
+      "description": "The URL of the Mesos master. The format is a comma-delimited list of of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard URLs like http://localhost are also acceptable.",
+      "type": "string"
+    },
+    "marathon/cpus": {
+      "description": "CPU shares to allocate to each Marathon instance.",
+      "type": "number",
+      "minimum": 0.0,
+      "default": 1.0
+    },
+    "marathon/mem": {
+      "description": "Memory (MB) to allocate to each Marathon instance.",
+      "type": "number",
+      "minimum": 0.0,
+      "default": 512.0
+    },
+    "marathon/instances": {
+      "description": "Number of Marathon instances to run.",
+      "type": "integer",
+      "minimum": 0,
+      "default": 1
+    },
+    "marathon/artifact_store": {
+      "description": "URL to the artifact store. Examples: \"hdfs://localhost:54310/path/to/store\", \"file:///var/log/store\". For details, see the artifact store docs: https://mesosphere.github.io/marathon/docs/artifact-store.html.",
+      "type": "string"
+    },
+    "marathon/checkpoint": {
+      "description": "Enable checkpointing of tasks. Requires checkpointing enabled on slaves. Allows tasks to continue running during mesos-slave restarts and Marathon scheduler failover.",
+      "type": "boolean",
+      "default": true
+    },
+    "marathon/executor": {
+      "description": "Executor to use when none is specified.",
+      "type": "string"
+    },
+    "marathon/failover-timeout": {
+      "description": "The failover_timeout for Mesos in seconds. If a new Marathon instance has not re-registered with Mesos this long after a failover, Mesos will shut down all running tasks started by Marathon. Requires checkpointing to be enabled.",
+      "type": "integer",
+      "minimum": 0,
+      "default": 604800
+    },
+    "marathon/framework-name": {
+      "description": "The framework name to register with Mesos.",
+      "type": "string"
+    },
+    "marathon/ha": {
+      "description": "Runs Marathon in HA mode with leader election. Allows starting an arbitrary number of other Marathons but all need to be started in HA mode. This mode requires a running ZooKeeper.",
```

Comment:
`HA` -> `high availability (HA)` (no brevity requirement here and it may not be intuitive to people what HA means)